docs/release_process.rst

```diff
@@ -0,0 +1,121 @@
+=====================
+LORE Release Workflow
+=====================
+
+Quick Start
+-----------
+
+prerequisites:
+  - ``git-release-notes`` - https://www.npmjs.com/package/git-release-notes
+  - ``git-flow`` - https://github.com/nvie/gitflow
+
+.. code-block:: bash
+
+    git fetch
+    git checkout master
+    git pull
+    git flow release start 0.10.0
+    vi lore/settings.py # change version number
+    vi util/release_notes_rst.ejs # change version temporarily -- don't commit!
+    # find the hashes for the first and last commits to include
+    git-release-notes v0.9.0..master util/release_notes_rst.ejs
+    vi RELEASE.rst # paste results of prior command at top of file
+    git add RELEASE.rst lore/settings.py # don't commit change to util/release_notes_rst.ejs
+    git tag v0.10.0
+    git commit -m "Release 0.10.0"
+    # Wait for all checkboxes to be checked
+    # Check with Dev-Ops buddy whether migrations need to be run for this release
+    git flow release publish 0.10.0
+    # Send release notification to LORE mailing list - odl-lore-announce@mit.edu
+
+2. Cut the Release Candidate
+----------------------------
+
+``git flow release start 0.10.0``
+
+3. Generate the Release Notes
+-----------------------------
+Create the release notes using a utility called ``git-release-notes``.
+https://www.npmjs.com/package/git-release-notes
+
+- Update the ``VERSION`` number in ``lore/settings.py`` following Semantic
+  numbering practice.
```

Comment:
I think updating the version number should be a separate step from generating the release notes.